checker/manual/annotating-libraries.tex

```diff
@@ -603,6 +609,29 @@ \subsection{Limitations\label{stub-limitations}}
   \begin{Verbatim}
           void init(@Nullable java.security.SecureRandom random);
    \end{Verbatim}
+\item
+   Annotations on types that are inner classes must be written before the
+   package if the type is written as a
+   fully annotated type, or before the inner class name if the type is
+   written as a simple type. For example,
+
+\begin{Verbatim}
+        ProcessBuilder redirectError(@NonNull java.lang.ProcessBuilder.Redirect destination);
```

Comment:
Can we mark the cases in the manual that violate Java 8 syntax, to highlight the discrepancies to source code?
Also, for each such mismatch, can we file an issue against the stub parser and reference the issue here.
(In Java 8 syntax, the example should be "java.lang.ProcessBuilder.@NonNull Redirect".)